### 다음 물음에 답하는 코드를 작성하시오. 

1). A학급의 학생 신장을 조사해 보니 다음과 같다:

161.5 <br>
155.9 <br>
168.7 <br>
163.1 <br>
170.8 <br>
167.2 <br>

위 자료를 리스트로 정리해본다.

In [1]:
listA = [161.5, 155.9, 168.7, 163.1, 170.8, 167.2]
listA

[161.5, 155.9, 168.7, 163.1, 170.8, 167.2]

리스트와 for 반복문을 사용해서 평균 신장을 구해본다.

In [2]:
sumA = 0
for A in listA:
    sumA = sumA + A
sumA/len(listA)

164.53333333333333

2). B학급의 학생 신장을 조사해 보니 다음과 같다:

172.5 <br>
162.9 <br>
161.7 <br>
160.1 <br>
161.8 <br>
159.2 <br>

A학급과 같은 방법으로 평균 신장을 구해본다.

In [3]:
listB = [172.5, 162.9, 161.7, 160.1, 161.8, 159.2]
sumB = 0
for B in listB:
    sumB = sumB + B
sumB/len(listB)
listB

[172.5, 162.9, 161.7, 160.1, 161.8, 159.2]

3). A학급과 B학급 신장 자료를 통합한 리스트 AB를 만든다. (이어 붙이기) <br>

이 통합 리스트 AB를 가지고 평균을 구한다.

In [4]:
listAB = listA + listB
listAB

[161.5,
 155.9,
 168.7,
 163.1,
 170.8,
 167.2,
 172.5,
 162.9,
 161.7,
 160.1,
 161.8,
 159.2]

4). 통합 리스트 AB를 정렬해 본다.

In [5]:
listAB.sort()
listAB

[155.9,
 159.2,
 160.1,
 161.5,
 161.7,
 161.8,
 162.9,
 163.1,
 167.2,
 168.7,
 170.8,
 172.5]

5). 통합 리스트 AB에서 가장 작은 값을 가져오시오.

In [6]:
listAB[0]

155.9

6). 통합 리스트 AB에서 가장 큰 값을 가져오시오.

In [7]:
listAB[len(listAB)-1]

172.5

7). 통합 리스트 AB를 사용하여 신장이 165이하인 학생들 만의 리스트 C를 만든다.

In [12]:
listC = list()
for AB in listAB:
    if(AB <= 165.0):
        listC.append(AB)
listC

[155.9, 159.2, 160.1, 161.5, 161.7, 161.8, 162.9, 163.1]

리스트 C의 평균을 구하여라.

In [13]:
sumC = 0
for C in listC:
    sumC = sumC + C
sumC/len(listC)

160.775

8). 통합 리스트 AB를 사용하여 신장이 160이상 170이하인 학생들 만의 리스트 D를 만든다.

In [14]:
listD = list()
for AB in listAB:
    if(160 <= AB and AB <= 170):
        listD.append(AB)
listD

[160.1, 161.5, 161.7, 161.8, 162.9, 163.1, 167.2, 168.7]

리스트 D의 평균을 구하여라.

In [15]:
sumD = 0
for D in listD:
    sumD = sumD + D
sumD/len(listD)

163.375